README.md

```diff
@@ -1,17 +1,121 @@
+
+## Client API Reference
+
+Visit the [full API documentation](https://js-libjuju.github.io/) for detailed information on the Client API.
+
+## Facade API Reference
+
+Defailed Facade documentation van be found by visiting the [full API documentation](https://js-libjuju.github.io/) or you can visit the facade source directly using the links below.
+
+- [action-v6.ts](api/facades/action-v6.ts)
+- [admin-v3.ts](api/facades/admin-v3.ts)
+- [all-model-watcher-v2.ts](api/facades/all-model-watcher-v2.ts)
+- [all-watcher-v1.ts](api/facades/all-watcher-v1.ts)
+- [annotations-v2.ts](api/facades/annotations-v2.ts)
+- [application-offers-v2.ts](api/facades/application-offers-v2.ts)
+- [application-v12.ts](api/facades/application-v12.ts)
+- [backups-v2.ts](api/facades/backups-v2.ts)
+- [block-v2.ts](api/facades/block-v2.ts)
+- [bundle-v1.ts](api/facades/bundle-v1.ts)
+- [bundle-v4.ts](api/facades/bundle-v4.ts)
+- [charms-v2.ts](api/facades/charms-v2.ts)
+- [client-v2.ts](api/facades/client-v2.ts)
+- [cloud-v1.ts](api/facades/cloud-v1.ts)
+- [cloud-v2.ts](api/facades/cloud-v2.ts)
+- [cloud-v3.ts](api/facades/cloud-v3.ts)
+- [cloud-v4.ts](api/facades/cloud-v4.ts)
+- [cloud-v5.ts](api/facades/cloud-v5.ts)
+- [cloud-v7.ts](api/facades/cloud-v7.ts)
+- [controller-v3.ts](api/facades/controller-v3.ts)
+- [controller-v4.ts](api/facades/controller-v4.ts)
+- [controller-v5.ts](api/facades/controller-v5.ts)
+- [controller-v6.ts](api/facades/controller-v6.ts)
+- [controller-v7.ts](api/facades/controller-v7.ts)
+- [controller-v8.ts](api/facades/controller-v8.ts)
+- [controller-v9.ts](api/facades/controller-v9.ts)
+- [credential-manager-v1.ts](api/facades/credential-manager-v1.ts)
+- [firewall-rules-v1.ts](api/facades/firewall-rules-v1.ts)
+- [high-availability-v2.ts](api/facades/high-availability-v2.ts)
+- [image-manager-v2.ts](api/facades/image-manager-v2.ts)
+- [image-metadata-manager-v1.ts](api/facades/image-metadata-manager-v1.ts)
+- [key-manager-v1.ts](api/facades/key-manager-v1.ts)
+- [machine-manager-v6.ts](api/facades/machine-manager-v6.ts)
+- [metrics-debug-v2.ts](api/facades/metrics-debug-v2.ts)
+- [model-config-v2.ts](api/facades/model-config-v2.ts)
+- [model-generation-v4.ts](api/facades/model-generation-v4.ts)
+- [model-manager-v2.ts](api/facades/model-manager-v2.ts)
+- [model-manager-v3.ts](api/facades/model-manager-v3.ts)
+- [model-manager-v4.ts](api/facades/model-manager-v4.ts)
+- [model-manager-v5.ts](api/facades/model-manager-v5.ts)
+- [model-manager-v8.ts](api/facades/model-manager-v8.ts)
+- [payloads-v1.ts](api/facades/payloads-v1.ts)
+- [pinger-v1.ts](api/facades/pinger-v1.ts)
+- [resources-v1.ts](api/facades/resources-v1.ts)
+- [spaces-v6.ts](api/facades/spaces-v6.ts)
+- [sshclient-v2.ts](api/facades/sshclient-v2.ts)
+- [storage-v6.ts](api/facades/storage-v6.ts)
+- [subnets-v4.ts](api/facades/subnets-v4.ts)
+- [user-manager-v1.ts](api/facades/user-manager-v1.ts)
+- [user-manager-v2.ts](api/facades/user-manager-v2.ts)
```

Comment:
Are these likely to get out of date?

Reply:
I wonder if there is a single index generated by the docs?

Reply:
Nope this file is automatically generated when the facade is so it'll mirror the actual api.